example/config.yml

```diff
@@ -1,5 +1,5 @@
 backend:
-  name: test-repo
+  name: file-system
 
```

Comment:
The default config file will allow for devServer (webpack) to start testing immediately on local branch checkout. The test-repo backend is not really needed once this solution is in place.